ServerCore/Pages/Resources/Default/FAQPartial.cshtml

```diff
@@ -1,4 +1,4 @@
-﻿@{
+﻿﻿@{
```

Comment:
Nit: This file and the next two can be removed

Reply:
Done